scripts/redis_crash_handler.py

```
@@ -0,0 +1,1144 @@
+    ]
+    
+    missing_vars = [var for var in required_env_vars if not os.environ.get(var)]
+    if missing_vars:
+        print(f"❌ Error: Missing required environment variables: {', '.join(missing_vars)}", file=sys.stderr)
+        sys.exit(1)
+    
+    # Get environment variables
+    omnistrate_url = os.environ['OMNISTRATE_API_URL']
+    omnistrate_user = os.environ['OMNISTRATE_USERNAME']
+    omnistrate_pass = os.environ['OMNISTRATE_PASSWORD']
+    service_id = os.environ['OMNISTRATE_SERVICE_ID']
```

Ruff (0.14.14)

[error] Local variable `service_id` is assigned to but never used

Remove assignment to unused variable `service_id`

(F841)